### PR TITLE
Improve build time

### PR DESCRIPTION
**Pull request by sansmoraxz**
I have noticed that the majority of the bottleneck is of git clones, taking insanely long if there are a lot of languages. This PR will parallelize the process.

Also the cloned folder list are already available. No need to explicitly pass it.

### CODE REVIEW

**Comment by sansmoraxz** [comment]
Wierd. The js based libraries are missing `src/parser.c`

**Comment by grantjenks** [comment]
I don’t see any speed up in the CI build time. Looks like it’s still 3h11m. I do see the parallelization but I don’t think this is the slow part.

**Comment by sansmoraxz** [comment]
OK the Linux builds are taking long. That auto flag is building for all architectures and versions. Will look into it.

Probably matrix items can be increased.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,43 +9,120 @@ on:
...
+          # additional archs for linux
+          - os: ubuntu-latest
+            CIBW_ARCHS: aarch64
+          - os: ubuntu-latest
+            CIBW_ARCHS: ppc64le
+          - os: ubuntu-latest
+            CIBW_ARCHS: s390x
+          # additional archs for windows
+          - os: windows-latest
+            CIBW_ARCHS: ARM64
```
Comment (sansmoraxz) [inline .github/workflows/release.yml:94]: IMO this seems a bit complex to manage, due to the compatibility matrix.

Shall we have separate configs for different OS?

**Comment by sansmoraxz** [comment]
Also the biggest bottlenecks are building for QEMU based `ppc64le`, `s390x` and `aarch64` distros.

**Pull request closed without merging**